src/content/community/releases/_index.en.md

```diff
@@ -19,6 +19,8 @@ weight = 40
 * Submariner gateway pods now skip invoking cable engine cleanup during termination, as this is handled by the route agent
   during gateway migration.
 * Fixed issue which caused the IPsec pluto process to crash when the remote endpoint was unstable.
+* The `Allocated` status condition type for Globalnet resources now reflects only the last status observed to conform with
```

Comment:
Looks fine. What about the following?

The status condition type "Allocated" for Globalnet resources now adheres to the intended design of status conditions in Kubernetes by reflecting only the latest observed status.